packages/widgetbook/lib/src/addons/zoom_addon/zoom_addon.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+import '../../fields/fields.dart';
+import '../common/common.dart';
+
+class ZoomAddon extends WidgetbookAddon<double> {
+  ZoomAddon({double initialZoom = 1.0})
+      : assert(initialZoom > 0.5 && initialZoom <= 3.0),
+        super(
+          name: 'Zoom',
+          initialSetting: initialZoom,
+        );
+
+  @override
+  List<Field> get fields => [
+        ZoomControlField(
+          name: 'zoom',
+          initialValue: initialSetting,
+        ),
```

Comment:
The fields should reflect primitive types, or simple ones. So I am not a fan of creating `ZoomControlField`, I would suggest we use the existing `DoubleSliderField` instead.

Reply:
This PR has been updated accordingly.